ripples/multi_session.py

```
@@ -25,403 +32,1150 @@
-    # TODO: add in statistics
 
+def plot_cluster_characteristics(
+    all_data: pd.DataFrame,
+    area_map: dict,
+    characteristics: List[str],
+    y_labels: List[str],
+) -> None:
+    groups = {"E": "Excitatory", "I": "Inhibitory"}
+    for area in area_map:
+        dataframe = all_data[all_data["region"] == area]
+        dataframe = dataframe[dataframe["quality_metrics"] == True]
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if dataframe["quality_metrics"]:` for truth checks

Replace with `dataframe["quality_metrics"]`

(E712)